README.md

```diff
@@ -56,24 +56,43 @@ and [contributing guidelines](CONTRIBUTING.md).
 
 ## Quickstart
 
-Run this command to start FerretDB with PostgreSQL:
+Run this command to start FerretDB with PostgreSQL, ensure to update `<username>` and `<password>`:
 
 ```sh
 docker run -d --rm --name ferretdb -p 27017:27017 --platform linux/amd64 \
+  -e POSTGRES_USER=<username> \
+  -e POSTGRES_PASSWORD=<password> \
+  -v ./data:/var/lib/postgresql/data \
```

Comment:
Actually, let's keep the README mostly the way it was – it is a README, after all, it should be short and to the point. But the evaluation images documentation should mention the volume, etc.

Reply:
Let's only update changes caused by enforcing authentication.